src/test/stateful_test_utils.hpp

```diff
@@ -76,6 +86,32 @@ class MockIVariableStateWithData : public MockIVariableState {
     }
 };
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-function"
+static void setRequestSequenceId(tensorflow::serving::PredictRequest* request, uint64_t sequence_id) {
+    auto& input = (*request->mutable_inputs())[SEQUENCE_ID_INPUT];
+    input.add_uint64_val(sequence_id);
+}
+
+static void setRequestSequenceControl(tensorflow::serving::PredictRequest* request, uint32_t sequence_control) {
```

Comment:
Can we use camelCase in parameter names? (sequence_control -> sequenceControlInput)